ONSdigital/ras-frontstage
# Ras-1035: Spike log based labelled metric for frontstage send message

# What and why?
This is a PR is part of a spike to look at adding metrics for secure messaging when a message is received from frontstage.

# How to test?
- Run acceptance tests
- Deploy this and https://github.com/ONSdigital/ras-secure-message/pull/408 to your dev env
- Send a variety of messages in frontstage
- In GCP goto Monitoring -> Metrics explorer -> Select metrics -> Unspecified resource > Logs-based metrics > logging/user/Frontstage-secure-messages-sent
- In he filter select 'InternalUser' with a value of 'false'
- Ensure the timescale is correct for when you sent the logs
- You should see activity on the graph

# Jira
https://jira.ons.gov.uk/browse/RAS-1035
